machine.go

```diff
@@ -464,8 +464,17 @@ func (m *Machine) startup(command string, extracontent [][2]string) (int, error)
 	if mergedUsrSystem() {
 		prefix = "/usr"
 	}
-	w.CopyFile(prefix + "/lib/x86_64-linux-gnu/libresolv.so.2")
-	w.CopyFile(prefix + "/lib/x86_64-linux-gnu/libc.so.6")
+
+	if _, err := os.Stat(prefix + "/lib/x86_64-linux-gnu/libresolv.so.2"); err == nil {
+		w.CopyFile(prefix + "/lib/x86_64-linux-gnu/libresolv.so.2")
+	} else {
+		w.CopyFile(prefix + "/lib/libresolv.so.2")
+	}
+	if _, err := os.Stat(prefix + "/lib/x86_64-linux-gnu/libc.so.6"); err == nil {
+		w.CopyFile(prefix + "/lib/x86_64-linux-gnu/libc.so.6")
+	} else {
+		w.CopyFile(prefix + "/lib/libc.so.6")
+	}
```

Comment:
this could be a lot nicer if we had a list of reasonable prefixes (and probably a list of files to copy) or a dedicated function to find the file in the various paths.. For the original code what wasn't done as it was just two CopyFile calls but this change adds a lot of extra noise

Reply:
Can we just `readlink` it? AFAIK `/lib/libc.so.6` and `/lib64/ld-linux-x86-64.so.2` are well-known paths that are always there.

Reply:
`libc.so.6` is not a well-known path; it's just another library.  The linker is a well known-path per architecture as defined by  https://sourceware.org/glibc/wiki/ABIList . 

On my debian system:
```
rwxrwxrwx 1 root root 32 Oct 31 23:31 /lib64/ld-linux-x86-64.so.2 -> /lib/x86_64-linux-gnu/ld-2.29.so
```

So i guess one could readlink it and work out a reasonable prefix from that; Though i don't know if on other distributions it's also a symlink to the same prefix as where all other libraries live ?

Reply:
> `libc.so.6` is not a well-known path; it's just another library

True, sorry, confused it with `ld-linux.so.2`.

> it's also a symlink to the same prefix as where all other libraries live ?

I guess it's very likely to be this way.

Reply:
Maybe we could use `ldconfig -p` or something similar to find libraries?

Reply:
> So i guess one could readlink it and work out a reasonable prefix from that; Though i don't know if on other distributions it's also a symlink to the same prefix as where all other libraries live ?

It works for ArchLinux and Manjaro (where `/lib`, `/lib64` and `/usr/lib64` are all symlinks to `/usr/lib`).

Just to be sure, I had a look at Ubuntu 18.04 LTS, it's also the case

> I guess it's very likely to be this way.

So you mean it's also true for Fedora.

Reply:
```
$ docker run -it fedora ls -l /lib64/ld-linux-x86-64.so.2
lrwxrwxrwx 1 root root 10 Jun  6  2019 /lib64/ld-linux-x86-64.so.2 -> ld-2.29.so
$ docker run -it fedora ls -l /lib64/libc.so.6
lrwxrwxrwx 1 root root 12 Jun  6  2019 /lib64/libc.so.6 -> libc-2.29.so
```
So yeah i would expect the "what directory does the linker live" to be a reasonableish guess for most libraries.

Reply:
I implemented it this way in the updated PR.